rpm/setools.spec

```diff
@@ -20,21 +20,20 @@
 # TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 # SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-%global sepol_ver 3.1
-%global selinux_ver 3.1
+%global sepol_ver 3.6
+%global selinux_ver 3.6
```

Comment:
The other packages in the bunch have version `3.7` here, can this cause issues?

Reply:
Shouldn't cause problem, it's only the minimum version dependency.